lte/gateway/c/core/oai/test/gtpv2-c/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_cc//cc:defs.bzl", "cc_test")
+
+package(default_visibility = ["//visibility:private"])
+
+cc_test(
+    name = "gtpv2_fteid_test",
+    srcs = [
+        "test_fteid.cpp",
+    ],
+    deps = [
+        "//lte/gateway/c/core",
```

Comment:
nit/personal preference: I often use full text search for finding usages of a target. For this it helps to have targets fully specified, i.e., "//lte/gateway/c/core:core". (But this workflow might become obsolete if/when we improve the vscode bazel support).

Reply:
I agree, I appreciate the fully specified label - thanks for the detection!

Reply:
So with the defaults provided right now in buildifer, it actually has stripped all of these labels and I cannot re-add them and pass buildifier.  We'll need to change buildifier config with ?`.bazelrc` or some such?

Reply:
To be honest, I am a little confused ...
* can not find this in https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md
* even if I set the buildifier to "warn", `:core` is removed
* always thought that we used the fully qualified pattern in python - but the buildifier did remove these as well

I do not mind to ignore this for now and have a look at a later time.

Reply:
ah, its `mode = "check"` - but I do not see options here to disable single format issues

Reply:
there does not seem to be a good solution :/ https://github.com/bazelbuild/buildtools/issues/806